modules/client/test/integration/client/decoding.test.ts

```diff
@@ -221,24 +206,10 @@ describe("Client", () => {
     it("Should try to decode an encoded update metadata action with the withdraws decoder and return an error", async () => {
       const context = new Context(contextParamsLocalChain);
       const client = new Client(context);
-      const params: IMetadata = {
-        name: "New Name",
-        description: "New description",
-        avatar: "https://theavatar.com/image.jpg",
-        links: [
-          {
-            url: "https://discord.com/...",
-            name: "Discord",
-          },
-          {
-            url: "https://twitter.com/...",
-            name: "Twitter",
-          },
-        ],
-      };
+
       const updateMetadataAction = await client.encoding.updateMetadataAction(
         "0x1234567890123456789012345678901234567890",
-        params,
```

Comment:
Why do we remove this feature?  Now the SDK use will have to encode, upload, pin, etc on his own?

Reply:
The goal of the SDK is to act as a one stop shop, now to simply expose low level functions so that developers need to learn about many internal operations

Reply:
a mistake from where I started to refactor some stuff. 
It will be added back